notebooks/test_deployed_model.ipynb

```diff
@@ -0,0 +1 @@
f2c5673d4562ba86ad3\", \"sha256:ecc7193fff7741ced3db1f760666c8454d6664956288c54d1b49613b987a42f4\"], \"index\": \"pypi\"}, \"torchvision\": {\"version\": \"==0.10.0\", \"hashes\": [\"sha256:20a57ce42fa20c26d800c65d5b88dbaaa115a01f4f5623d41abfb182b854f199\", \"sha256:4796e1fb2995c6e495c5ea4e2b0fe0e4be44bd9416ef4a1349c1a406675cbdee\", \"sha256:487bbfd89575a52cd18bca8a33e24c373570e060f801265051c3a0aafc769720\", \"sha256:576d7b070f25cbfc78a710960fd8fa6d3961d640db05f7ace69d9a3e5bbf754a\", \"sha256:6b917d4762deaaa4c0cdd106403ea8384a1fdf93de424097bd71f3ebfdc76b41\", \"sha256:6d69bf15a6e885e3b8c674a524bc6b53016879cb8b0fd8537327edd2d243cab1\", \"sha256:9597da592f76d22d9e80a4a072294e093f8c3a06c404f3ff237f359b9225e097\", \"sha256:a30466893a5b97073c992859f3645e3e1f41daf2c1b4db6cb2ac8ec7d0e1f6bc\", \"sha256:a405d968e7e94c0744870eef31c977635e2123b0b46becc1461a28b7c27d3c0c\", \"sha256:aa709fa63896f93e03a03976230a51050fcd5f1b45cf663f62d91b7eaaf8ac09\", \"sha256:ad24107b2ed0ccc372af92822f1f8f5530907b6fb7520a08195cf0bb07446923\", \"sha256:ae9606cc248de9b3a077710529b11c57315d2914c8ee3099fbd93a62f56a1661\", \"sha256:bb04708354fb6d639f6e47d8066b0d546fbe0a3a68685cf8d413a6370c8f63ad\", \"sha256:ce2aae5567522f2a877c6334796721af07c164e94ff75876821fadb3310cfe7e\", \"sha256:d5c80cf058c0ecb010a97dc71225f5274e45c596bfad7505e0d000abcccb7063\", \"sha256:da65af25b51caf43327ecb3ccf550eedfd62d1f73511db44370b4b9522569b8d\", \"sha256:f26c5ba220445ee8e892033234485c9276304874e87ec9d5146779167be3148d\", \"sha256:f68291559a4cf1245c95efc5e47ebe158819aceec4e1f585d2fe133cd2c9d8e8\"], \"index\": \"pypi\"}, \"typing-extensions\": {\"version\": \"==3.7.4.3\", \"hashes\": [\"sha256:7cb407020f00f7bfc3cb3e7881628838e69d8f3fcab2f64742a5e76b2f841918\", \"sha256:99d4073b617d30288f569d3f13d2bd7548c3a7e4c8de87db09a9d29bb3a4a60c\", \"sha256:dafc7639cde7f1b6e1acc0f457842a83e722ccca8eef5270af2d74792619a89f\"]}, \"urllib3\": {\"version\": \"==1.26.6\", \"hashes\": [\"sha256:39fb8672126159acb139a7718dd10806104dec1e2f0f6c88aab05d17df10c8d4\", \"sha256:f57b4c16c62fa2760b7e3d97c35b255512fb6b59a259730f36ba32ce9f8e342f\"], \"markers\": \"python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'\"}, \"werkzeug\": {\"version\": \"==2.0.1\", \"hashes\": [\"sha256:1de1db30d010ff1af14a009224ec49ab2329ad2cde454c8a708130642d579c42\", \"sha256:6c1ec500dcdba0baa27600f6a22f6333d8b662d22027ff9f6202e3367413caa8\"], \"markers\": \"python_version >= '3.6'\"}, \"wheel\": {\"version\": \"==0.37.0\", \"hashes\": [\"sha256:21014b2bd93c6d0034b6ba5d35e4eb284340e09d63c59aef6fc14b0f346146fd\", \"sha256:e2ef7239991699e3355d54f8e968a21bb940a1dbf34a4d226741e64462516fad\"], \"markers\": \"python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'\"}, \"wrapt\": {\"version\": \"==1.12.1\", \"hashes\": [\"sha256:b62ffa81fb85f4332a4f609cab4ac40709470da05643a082ec1eb88e6d9b97d7\"]}}, \"develop\": {}}", "dependency_resolution_engine": "pipenv"}, "nbformat": 4, "nbformat_minor": 4}
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cluster_url = "https://api.zero.massopen.cloud:6443"
</pre><p>This should be updated to the smaug cluster url</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AICoE/elyra-aidevsecops-tutorial/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>